pkg/networkmanager/networking.css

```diff
@@ -290,6 +290,15 @@ h1 .onoff-ct {
 #firewall-heading,
 #networking-graphs {
     background: var(--color-white);
+}
+
+#networking-graphs {
+    margin: 0;
+    /* Bootstrap uses 20px padding for row children. Compensate. */
+    padding: 1rem calc(3rem - 20px);
+}
```

Comment:
Same for this, is this an independent fix, or somehow fallout from the scss transition and css variable scoping?